src/components/custom/LogoDisplay.js

```diff
@@ -0,0 +1,60 @@
+import { Box } from "@mui/material";
+import { DNALogo } from 'logojs-react';
+
+export default function LogoDisplay({data}) {
+
+
+        // Pull out each predicted operator and apply validations
+    const operators = [];
+    const op_length = data[0]["Predicted operator"].length;
+
+    data.forEach(item => {
+        const op = item["Predicted operator"].toUpperCase()
+        if (op.length === op_length && op.match(/^[ATCG]*$/)){
+            operators.push(op);
+            }
+    })
+
+        // Create the base pair frequency matrix
+    const num_ops = operators.length
+    const matrix = []
+
+        // define the position and initialize the start
+    for (let i = 0; i < operators[0].length; i++){
+        var base = [0,0,0,0];
```

Comment:
Can you swap this over to let? Var is a bit outdated

Reply:
Resolved in latest commit